### PR TITLE
qa: Wip cherry pick 4tamil ceph-ansible

### PR DESCRIPTION
**Pull request by yuriw**

### CODE REVIEW

File: qa/suites/ceph-ansible/smoke/basic/2-config/ceph_ansible.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        ceph_conf_overrides:                                                                                                                                                                                
+          global:                                                                                                                                                                                           
+            osd default pool size: 2                                                                                                                                                                        
+            mon pg warn min per osd: 2                                                                                                                                                                      
+        ceph_dev: true                                                                                                                                                                                      
+        ceph_dev_branch: jewel
```
Comment (vasukulkarni) [inline qa/suites/ceph-ansible/smoke/basic/2-config/ceph_ansible.yaml]: This should be kraken

**Comment by yuriw** [comment]
tests => http://pulpito.front.sepia.ceph.com:80/yuriw-2017-01-18_17:47:16-ceph-ansible-jewel-distro-basic-vps/

File: qa/suites/ceph-ansible/smoke/basic/2-config/ceph_ansible.yaml (hunk 1 above)
Comment (vasukulkarni) [inline qa/suites/ceph-ansible/smoke/basic/2-config/ceph_ansible.yaml]: actually should be removed so that it points to ceph branch.

**Comment by yuriw** [comment]
test run => http://pulpito.front.sepia.ceph.com:80/yuriw-2017-01-19_18:39:51-ceph-ansible-kraken-distro-basic-vps/

**Pull request merged**